.travis.yml

```diff
@@ -1,5 +1,10 @@
 sudo: required
+language: python
 dist: trusty
```

Comment:
we should probably update this to build on xenial and the env to use xenial and drop trusty and precise. Could you update that?